[2023-06-10T02:18:42.008+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: second_dag.first_task scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-06-10T02:18:43.162+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: second_dag.first_task scheduled__2023-05-14T00:00:00+00:00 [queued]>
[2023-06-10T02:18:43.164+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 6
[2023-06-10T02:18:44.169+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): first_task> on 2023-05-14 00:00:00+00:00
[2023-06-10T02:18:44.312+0000] {standard_task_runner.py:57} INFO - Started process 3652 to run task
[2023-06-10T02:18:44.585+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'second_dag', 'first_task', 'scheduled__2023-05-14T00:00:00+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpmb4a7wwk']
[2023-06-10T02:18:44.602+0000] {standard_task_runner.py:85} INFO - Job 263: Subtask first_task
[2023-06-10T02:18:53.312+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-10T02:18:53.680+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
